mk/darwin.mk

```
@@ -11,7 +11,7 @@ IOS_SDK_VERSION="9.0"
 
 else
 
-ifeq ($(XCODE_VERSION_MAJOR),11)
+ifeq ($(XCODE_VERSION_MAJOR),12)
```

```suggestion
ifeq ($(shell test $(XCODE_VERSION_MAJOR) -gt 10;echo $$?),0)
```

this way it will handle xcode11 and xcode12 (and beyond!)